### PR TITLE
Algebra and UniV3 config 

### PR DESCRIPTION
**Pull request by vminkov**
## Description

Uniswap v3 fees and Algebra liquidators config

### CODE REVIEW

File: packages/chains/src/bsc/addresses.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ const chainAddresses: ChainAddresses = {
   STABLE_TOKEN: BUSD,
   W_BTC_TOKEN: BTCB,
   ALGEBRA_SWAP_ROUTER: "0x327Dd3208f0bCF590A66110aCB6e5e6941A4EfA0",
+  ALGEBRA_UNI_PROXY: "0x6B3d98406779DDca311E6C43553773207b506Fa6",
```
Comment (carlomazzaferro) [inline packages/chains/src/bsc/addresses.ts:15]: We need 2 sets of these:

```
 GAMMA_ALGEBRA_SWAP_ROUTER
 GAMMA_ALGEBRA_UNI_PROXY
 GAMMA_UNISWAP_V3_SWAP_ROUTER
 GAMMA_UNISWAP_V3__UNI_PROXY
```

Comment (carlomazzaferro) [reply packages/chains/src/bsc/addresses.ts:15]: The first two are for QuickSwap (Algebra implementation with Gamma farms).

The other two are for Retro.Finance (UniswapV3 implementation with Gamma farms).

See: https://github.com/ionicprotocol/contracts/pull/25/files#diff-ce3f43bc3ed7bf85556bdf6bdf676be4c81449729227c5008989d7a8f824c5f0R30-R33

For the correct addresses for polygon

Comment (vminkov) [reply packages/chains/src/bsc/addresses.ts:15]: do we need these for BSC?

Comment (carlomazzaferro) [reply packages/chains/src/bsc/addresses.ts:15]: Not for the time being

File: packages/sdk/chainDeploy/helpers/liquidators/registry.ts [hunk 2]
```diff
@@ -17,25 +17,48 @@ export const configureLiquidatorsRegistry = async ({
...
+    } else {
+      console.log("no redemption strategies to configure in the liquidators registry");
+    }
   }
-  const matching = await liquidatorsRegistry.callStatic.pairsStrategiesMatch(strategies, inputTokens, outputTokens);
-  if (!matching) {
-    const tx = await liquidatorsRegistry._resetRedemptionStrategies(strategies, inputTokens, outputTokens);
+
+  {
+    // TODO check if same instead of updating every time
```
Comment (vminkov) [inline packages/sdk/chainDeploy/helpers/liquidators/registry.ts:45]: do we want to address this? I think such a check will bring the `LiquidatorsRegistryExtension` closer to the code size limit

**Review by carlomazzaferro — APPROVED** [review]

**Pull request merged**